src/Connection.cpp

```diff
@@ -140,7 +133,7 @@ int Connection::connectPoll(int fd, struct addrinfo* ai_ptr) {
         pollfd_t pollfds[n_fds];
         pollfds[0].fd = fd;
         pollfds[0].events = POLLOUT;
-        int max_timeout = 6;
+        int max_timeout = 3;
```

Comment:
Any reason for this?

Reply:
@mckelvin no particular reason，just think 6 as ratio is large。

Reply:
i will not insist if you think it's no okay。

Reply:
I'd prefer to keep it unchanged. If you think it's essential, I think you should also bump the version later to make users noticed.

Reply:
@mckelvin okay，i'll do it later if i think it's necessary after more observations.

Reply:
@mckelvin updated